src/main/java/com/demo/service/ParkingEventService.java

```diff
@@ -0,0 +1,41 @@
+package com.demo.service;
+
+import com.demo.model.ParkingEvent;
+import com.demo.repository.ParkingEventRepository;
+import lombok.Data;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.sql.Timestamp;
+import java.util.List;
+
+@Service
+@Data
+public class ParkingEventService {
+
+    @Autowired
+    private ParkingEventRepository parkingEventRepository;
+
+    public List<ParkingEvent> getAllEvents() {
+        return parkingEventRepository.findAll();
+    }
+
+    public void addNewEvent(double payment, Timestamp date) {
+        ParkingEvent parkingEvent = new ParkingEvent(payment, date);
+        parkingEventRepository.saveAndFlush(parkingEvent);
+    }
+
+    public double getPaymentSum() {
+        return parkingEventRepository.getPaymentSum();
+    }
+
+    public double getPaymentSumByDate(String year, String month, String day) {
+        String first = year + "-" + month + "-" + day;
```

Comment:
może masz jakiś inny pomysł na inną implementację tworzenia zakresu tych dat?

Reply:
Zostało to poprawione. Zastosowałem bibliotekę Joda-Time